packages/ui/src/components/Label/Label.tsx

```diff
@@ -17,17 +16,15 @@ export const Label: React.FC<PropsWithChildren<LabelProps>> = ({
 
   return (
     <Component
-      className={twMerge(
-        root({
-          fontWeight: {
-            initial: "xxl",
-            md: "xxl",
-          },
-          size,
-          className,
-          disabled,
-        })
-      )}
+      className={root({
+        fontWeight: {
+          initial: "xxl",
+          md: "xxl",
+        },
+        size,
+        className,
```

Comment:
@flozero is this possible to pass the class like `class: "foo"`?

Reply:
I will add this possibility in another pr to avoid too much brekaing changes. I have an issue for it